pkg/user/tx_client.go

```diff
@@ -190,6 +198,11 @@ func NewTxClient(
 		txTracker:           make(map[string]txInfo),
 		cdc:                 cdc,
 		gasEstimationClient: gasestimation.NewGasEstimatorClient(conn),
+		// By default, the client operates in trusted mode,
+		// which trusts the consensus node to provide accurate account information
+		// without additional verification.
+		// It is advised to use the consensus endpoint that you trust.
+		trustedMode: defaultTrustedMode, // Default to trusted mode is true
```

Comment:
given this is a bool, I would call it:
```suggestion
		trusted: true, // Default to trusting the consensus node
```
instead of trustedMode